kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/logs/TestLogCollector.java

```diff
@@ -141,7 +141,12 @@ private LogCollector defaultLogCollector() {
                 Constants.SERVICE.toLowerCase(Locale.ROOT),
                 Constants.JOB.toLowerCase(Locale.ROOT),
                 Kafka.RESOURCE_SINGULAR,
-                KafkaNodePool.RESOURCE_SINGULAR));
+                KafkaNodePool.RESOURCE_SINGULAR,
+                Constants.KROXYLICIOUS_KAFKA_PROXY_INGRESS_KIND.toLowerCase(Locale.ROOT),
```

Comment:
Given we don't have any usages outside of this one why don't we just have a lower case constant? Or a util method (fabric8 might even have one already) to convert to a valid kube name.

Reply:
Tested without `toLowerCase` for all elements and it works fine. So removed it